configs/fastscnn/fast_scnn_4x8_160k_lr0.12_cityscapes.py

```diff
@@ -0,0 +1,10 @@
+_base_ = [
+    '../_base_/models/fast_scnn.py', '../_base_/datasets/cityscapes.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_160k.py'
+]
+
+# Re-config the data sampler.
+data = dict(samples_per_gpu=4, workers_per_gpu=4)
```

Comment:
If we use 8 GPUs and 4 batch size per GPU, please rename config name `4x8` to `8x4`